### PR TITLE
Add athena tables DDL

### PR DESCRIPTION
**Pull request by alexduf**

### CODE REVIEW

File: athena-tables.sql [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    LOCATION
+    's3://<PROD_BUCKET>/fastly';
+
+
+CREATE OR REPLACE VIEW notification_received_prod AS
+  SELECT
+    "dateTime" "received_timestamp",
+    "regexp_extract"("querystring", 'notificationId=([\w-]+)', 1) "notificationId",
+    "regexp_extract"("querystring", 'platform=(\w+)', 1) "platform",
+    "regexp_extract"("querystring", 'provider=(\w+)', 1) "provider",
```
Comment (markrichardsg) [inline athena-tables.sql]: We don't need the provider any more.

File: athena-tables.sql [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    OUTPUTFORMAT
+    'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'
+    LOCATION
+    's3://<PROD_BUCKET>/fastly';
+
+
+CREATE OR REPLACE VIEW notification_received_prod AS
+  SELECT
+    "dateTime" "received_timestamp",
+    "regexp_extract"("querystring", 'notificationId=([\w-]+)', 1) "notificationId",
```
Comment (markrichardsg) [inline athena-tables.sql]: We might want to add a WHERE != unknown clause due to the bug that's been fixed, but still populating s3 with invalid events.

File: athena-tables.sql (hunk 1 above)
Comment (alexduf) [reply athena-tables.sql]: It's a field sent by the device, so I want to keep it

File: athena-tables.sql (hunk 2 above)
Comment (alexduf) [reply athena-tables.sql]: I'm not sure that's a concern the table should have

**Pull request merged**